### ISSUE TITLE
Stripe user agent not detected as bot

### ISSUE THREAD

**Issue by spdawson** [fnando/browser#258]
## Description

[Stripe](https://stripe.com) makes webhook calls using the following `User-Agent` header.

```text
Stripe/1.0 (+https://stripe.com/docs/webhooks)
```

This should be detected as a bot.

## How to reproduce

```ruby
Browser.new("Stripe/1.0 (+https://stripe.com/docs/webhooks)").bot?
```
## What do you expect

```ruby
true
```

## What happened instead

```ruby
false
```

## Software:

- Rails version: 4.2.7.rc1
- Browser gem version: 2.2.0

## Full backtrace

N/A

## Report

N/A

**Comment by nruth** [comment]
looks like this can be closed because #262 added this?

**Comment by fnando** [comment]
@nruth yep! :)
